BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.m

```diff
@@ -53,7 +53,7 @@ static BOOL BoxOperationStateTransitionIsValid(BOXAPIOperationState fromState, B
 
 @interface BOXAPIOperation()
 
-- (void)cancelConnection;
+- (void)cancelSessionTask;
```

Comment:
We can rename this if we want, but then it's again tech-specific. I left it because I was just thinking of it as an abstraction "cancelConnection" It's a network connection no matter what technology we use under the covers to do that.

Is there a real need for this to be specific for session task? Does it do sessionTask-unique things?